request.d.ts

```diff
@@ -0,0 +1,6 @@
+import { PinejsClientCore } from './core'
+import * as request from 'request'
+
+export default class PinejsClientRequest extends PinejsClientCore<PinejsClientRequest> {
+	_request: (params: request.Options) => Promise<any>
```

Comment:
As above, this (and the abstract method it's implementing) should have a more restrictive return type imo